smartclide-monitoring/src/main/java/de/atb/context/monitoring/models/GitMessage.java

```diff
@@ -37,7 +37,10 @@ public class GitMessage {
     String user;
     String repository;
     String branch;
+    String header;
+    String state;
     Integer noOfCommitsInBranch;
     Integer noOfPushesInBranch;
```

Comment:
`noOfCommitsInBranch`, `noOfPushesInBranch`, and `timestamp` are never used and can be removed.